README.md

```diff
@@ -1,15 +1,111 @@
-# Sourceloop Angular Boiler Plate
+#### ANGULAR BOILERPLATE
+[![Version](https://img.shields.io/badge/@angular/core-v14-brightgreen)](http://commitizen.github.io/cz-cli/)
 
-### Feedback
+<!-- DOCUMENTATION -->
 
-If you've noticed a bug or have a question or have a feature request, [search the issue tracker]([Issues · sourcefuse/angular-boilerplate · GitHub](https://github.com/sourcefuse/angular-boilerplate/issues)) to see if someone else in the community has already created a ticket. If not, go ahead and [make one](https://github.com/sourcefuse/angular-boilerplate/issues/new/choose)! All feature requests are welcome. Implementation time may vary. Feel free to contribute the same, if you can. If you think this extension is useful, please [star](https://help.github.com/en/articles/about-stars) it. Appreciation really helps in keeping this project alive.
+# Description
```

Comment:
It can't be h1 heading. Recheck the hierarchy in all your docs there are many mismatches. I'm refraining from adding comments everywhere.